src/spatialNavigation.js

```diff
@@ -291,6 +291,7 @@ class SpatialNavigation {
     this.enabled = false;
     this.nativeMode = false;
     this.throttle = 0;
+    this.isThrottledFastClicking = false;
```

Comment:
Could we use a more descriptive name for this flag? 

"Fast clicking" sounds like it's related to mouse-clicks, so I think it could be more clear.

- `throttleKeypress`
- `keystrokeThrottling`
..